modules/ROOT/pages/major-changes.adoc

```diff
@@ -9,6 +9,40 @@ This list is in reverse chronological order to keep recent changes at the top.
 
 // To add a new change here, see the template at the end of the file.
 
+== Switch to OCI updates
+
+Starting in Fedora 42, Fedora CoreOS will be updating through OCI images instead of the
+OSTree repo.
+Fedora Change request: https://fedoraproject.org/wiki/Changes/CoreOSOstree2OCIUpdates
+More discussion: https://github.com/coreos/fedora-coreos-tracker/issues/1823
+
+=== Planning
+
+This change has been introduced first in bootimages for the testing stream, for the Fedora 42 rebase. After a few releases
```

Comment:
```suggestion
This change will be introduced first in new disk images for the testing stream, as part of the rebase to Fedora 42. After a few releases
```

Reply:
reworded so next stream is in past tense and testing and stable will follow